soh/soh/Enhancements/randomizer/randomizer.cpp

```diff
@@ -65,194 +65,208 @@ const std::string Randomizer::NaviRandoMessageTableID = "RandomizerNavi";
+                                              "Riot Points",
+                                              "Rials",
+                                              "Riyals",
+                                              "Robux",
+                                              "Rubies",
+                                              "Rubles",
+                                              "Runite Ore",
+                                              "Rupees",
+                                              "Saint Quartz",
+                                              "Sapphires",
+                                              "Septims",
+                                              "Shekels",
+                                              "Shillings",
+                                              "Silver",
+                                              "Simoleons",
+                                              "Smackaroos",
+                                              "Social Credit",
+                                              "Souls",
+                                              "Spent Casings",
+                                              "Spice",
+                                              "Spondulicks",
+                                              "Spoons",
+                                              "Star Bits",
+                                              "Star Chips",
+                                              "Stars",
+                                              "Stones of Jordan",
+                                              "Store Credit",
+                                              "Strawbs",
+                                              "Studs",
+                                              "Super Sea Snails",
+                                              "Talent",
+                                              "Teef",
+                                              "Telecrystals",
+                                              "Tiberium",
+                                              "TokKul",
+                                              "Toys",
+                                              "Turnips",
+                                              "Upvotes",
+                                              "V-Bucks",
+                                              "Vespene Gas",
+                                              "Watts",
+                                              "Widgets",
+                                              "Woolongs",
+                                              "World Dollars",
+                                              "Wumpa Fruit",
+                                              "Won",
+                                              "Yen",
+                                              "Yuan",
+                                              "Zenny",
+                                              "Zorkmids" };
```

Comment:
This file is changing a lot because you removed a trailing comma here on `"Zorkmids"` which is causing clang-format to change the indenting. Please add back the trailing comma and reformat with clang.